docs/tutorials/core-from-scratch.ipynb

```diff
@@ -0,0 +1,369 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "0",
+   "metadata": {},
+   "source": [
+    "# Jaxoplanet core from scratch\n",
+    "\n",
+    "Inspired by the [autodidax tutorial](https://jax.readthedocs.io/en/latest/autodidax.html) from the JAX documentation, in this tutorial we work through implementing some of the core `jaxoplanet` functionality from scratch, to demonstrate and discuss the choices made within the depths of the codebase.\n",
+    "\n",
+    "For now, we only cover the included solver for Kepler's equation, but we hope to include some discussion of the light curve models in a later edition because those include some interesting tradeoffs and other details."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1",
+   "metadata": {},
+   "source": [
+    "## Solving Kepler's equation\n",
+    "\n",
+    "One core piece of infrastructure provided by `jaxoplanet` is a function to solve Kepler's equation\n",
+    "\n",
+    "$$\n",
+    "M = E - e \\sin(E)\n",
+    "$$\n",
+    "\n",
+    "for the eccentric anomaly $E$ as a function of the eccentricity $e$ and mean anomaly $M$.\n",
+    "There is a lot of literature dedicated to solving this equation efficiently and robustly, and we won't get into all the details here, but there are a few points we should highlight:\n",
```

Comment:
I don't have strong feelings but even some extremely short context as to why it's important to solve the Kepler equation might be nice? 

"There is a lot of literature dedicated to solving this equation efficiently and robustly _as it provides various geometric properties for the motion of a body orbiting a central object_,..."

Reply:
Done.